compiler/builtins/_internal_object.ts

```diff
@@ -367,6 +367,13 @@ export const __Porffor_object_set = (obj: any, key: any, value: any): any => {
   // write new value value (lol)
   Porffor.wasm.f64.store(entryPtr, value, 0, 4);
 
+  // note: v8 implements this with an internal accessor iirc? but indirect calls are really slow, so a strcmp
```

Comment:
I'm hesitant to add a strcmp to every object set tbh

Reply:
Hmm, how do you think this should be changed then? If I recall correctly, `foo["__proto__"]` should also do the same thing, so this can't be lifted to compile time

Reply:
What do you think about this new approach?